src/CoreBundle/private/library/classes/TCMSTableEditor/TCMSTableEditorEndPoint.class.php

```diff
@@ -2096,6 +2090,44 @@ public function DeleteIdConnectedRecordReferences()
         }
     }
 
+    protected function deleteMultiTableRecordReferences(string $tableName, string $id = ''): void
+    {
+        $databaseConnection = $this->getDatabaseConnection();
+
+        $query = 'SELECT `id` FROM `cms_field_type` WHERE `constname` = '.$databaseConnection->quote(TCMSFieldExtendedLookupMultiTable::FIELD_SYSTEM_NAME);
+        $fieldTypeId = $databaseConnection->fetchColumn($query);
```

Comment:
There are three queries in this method (all with name "query")?

One early return, one get all configured connections and the delete all data?

Maybe name them accordingly or even move to appropriately named method?

Reply:
moved one part out and renamed the other two.